pages/security/human-security/passwords/fr.md

```diff
@@ -0,0 +1,64 @@
+
+Peu importe l'option que vous choisissez, la clé est d'utiliser un gestionnaire de mots de passe.  Voici 2 astuces à garder en tête:
+
+* **Mot de passe maître:**  Lorsqu'on se sert d'un gestionnaire de mots de passe il est essentiel, primordial, de ne pas perdre le mot de passe maître.  Sans le mot de passe maître tous les autres mots de passe sont inaccessible.  Si vous pensez qu'il est possible d'oublier ce mot de passe maître, écrivez-le à quelque part et garder le bout de papier dans un lieu sûr.
+* **Copie de sauvegarde (backups):** Il est très important de faire des copies de sauvegarde de votre base de données des mots de passe.  Dans le cas des services infonuagiques, les sauvegardes sont  faits pour vous (il est quand même bien de faire une copie locale).  Dans le  cas des applications locales il suffit de faire une copie du fichier de données.
+
+Gestionnaires de mots de passe populaires:
+
+* KeePass et KeePassX (applications) sont 2 versions d'un même gestionnaire de mots de passe.  Les 2 sont très bien cotés.  Ces 2 outils utilisent le même format de fichier.  Ils roulent sur presque tous les ordinateurs.
+   * [Surveillance Self-defense / Guide pratique: utiliser KeePassXC](https://ssd.eff.org/fr/module/guide-pratique-utiliser-keepassxc)
+   * [KeePassX for Windows / Secure Password Manager](https://securityinabox.org/fr/guide/keepassx/windows)
+* LastPass (infonuagique)
+* 1Password (infonuagique et application)
+
+# Utiliser des mots de passe robustes
+
+Les mots de passe forts sont *générés de manière aléatoire*.  Mise-à-part les mots de passe pour votre appareil et de votre gestionnaire de mots de passe, tous vos mots de passe devraient être générés par le gestionnaire et devraient être d'une longueur minimale de 12 caractères.  Les mots de passe n'ont pas besoin d'être plus longs que 26 caractères.
+
+Les être humains ne créent pas de mots de passe robustes, mais les ordinateurs, eux, en créent d'excellents.  Laissez votre ordinateur générer vos mots de passe.
+
+Pour les mots de passe qu'il faut se souvenir, il y a plusieurs façons de générer de bons.  D'abord, il y a un guide [Créer des mots de passe robustes](https://ssd.eff.org/fr/module/créer-des-mots-de-passe-robustes).
+
+[Diceware](https://web.archive.org/web/20041012030451/www.gjldp.org/CHARENTAISES/article.php?id_article=4) ([Version originale en anglais](http://world.std.com/~reinhold/diceware.html)) est une façon efficace et ludique de créer des mots de passes faciles à se rappeler, mais aléatoires.  Il utilise des dès et une liste de mots.
+
+Une autre façon de créer des mots de passe forts est d'inventer une phrase loufoque que personne n'a jamais dit par auparavant et d'utiliser la première lettre ou deux de chaque mot pour construire son mot de passe tout en y mélangeant de la ponctuation et des chiffres.
```

Comment:
* In "que personne n'a jamais dit par auparavant", the "par" is superfluous
* I would replace "d'utiliser la première lettre ou deux de chaque mot" by "d'utiliser la première ou la seconde lettre de chaque mot"

Reply:
how about this?

"... d'utiliser un ou deux des premières lettres de chaque mot pour construire son mot de passe..."

Reply:
removed the superfluous "par"

Reply:
or this:

"... d'utiliser la ou les première(s) lettre(s) de chaque mot pour construire son mot de passe ..."

Reply:
or this:

"... d'utiliser la première lettre ou les deux premières lettres de chaque mot pour construire son mot de passe ..."

Reply:
I like "... d'utiliser la ou les première(s) lettre(s) de chaque mot pour construire son mot de passe ..." better